src/groups/general_unitary_groups.jl

```diff
@@ -136,6 +136,10 @@ function exp_lie!(::GeneralUnitaryMultiplicationGroup{TypeParameter{Tuple{4}},
     return q
 end
 
+function get_embedding(G::GeneralUnitaryMultiplicationGroup)
```

Comment:
Would it maybe make sense to define the embedding of an abstract group to be the embedding of the (stored within) manifold?

Reply:
I don't want to go that route. Basic dispatches are not sufficient to prevent incorrect dispatch, and I don't want to add more trait-based dispatch than necessary.

Reply:
Ok, that's why I asked :)